boilerplate/openshift/golang-osd-operator/csv-generate/common-generate-operator-bundle.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+#
+# Common script to generate OSD operator bundles for publishing to OLM. Copies appropriate files
+# into a directory, and composes the ClusterServiceVersion which needs bits and
+# pieces of our rbac and deployment files.
+#
+# Usage ./common-generate-operator-bundle.py -o OPERATOR_NAME -d OUTPUT_DIR -p PREVIOUS_VERSION -n GIT_NUM_COMMITS -c COMMIT_HASH -i HIVE_IMAGE
+
+import datetime
+import os
+import sys
+import yaml
+import shutil
+import argparse
+import string
+
+# This script will append the current number of commits given as an arg
+# (presumably since some past base tag), and the git hash arg for a final
+# version like: 0.1.189-3f73a592
+VERSION_BASE = "0.1"
```

Comment:
If we're running this script via the Makefile, there are [env vars](https://github.com/openshift/boilerplate/blob/69bf49e355dd9fe677a6c2d7a295949f6a2998e7/boilerplate/openshift/golang-osd-operator/project.mk#L9-L10) for this major/minor, which may be [overridden by some consumers](https://github.com/openshift/managed-velero-operator/blob/7ecdc226fd2d716c7d91de6cd0a6a712cefbc6e9/Makefile#L3). We should really use those instead of hardcoding.